packages/react-composites/src/composites/CallWithChatComposite/Strings.tsx

```diff
@@ -45,6 +45,14 @@ export interface CallWithChatCompositeStrings {
    * Only used with `mobileView` set to `true`.
    */
   moreDrawerSpeakerMenuTitle: string;
+  /**
+   * {@Link CallWithChatComposite} Mobile Control bar More drawer label for narrator
+   */
+  moreDrawerButtonLabel: string;
+  /**
+   * {@Link CallWithChatComposite} Mobile Control bar More drawer label for narrator
```

Comment:
The comment doesn't seem to match intent. This is used in the tooltip that is shown in the UI on hover, right?